framework-plugins/lisk-framework-forger-plugin/src/webhooks.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import axios, { AxiosError } from 'axios';
+import * as Debug from 'debug';
+
+import { Webhook } from './types';
+// eslint-disable-next-line new-cap
+const debug = Debug('plugin:forger:webhooks');
+
+interface HttpHeaders {
+	[key: string]: string;
+}
+
+interface BlockCreated {
+	readonly height: number;
+	readonly reward: string;
+	readonly forgerAddress: string;
+}
+
+interface BlockMissed {
+	readonly height: number;
+	readonly missedBlocksByAddress: { [key: string]: number };
+}
+
+interface NodeStatusChange {
+	readonly reason: string;
+}
+
+interface webHookPayload {
```

Comment:
uppercase

Reply:
Fixed. Is there an eslint rule for this maybe?